sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/StorageScope.cs

```diff
@@ -112,19 +112,19 @@ public static async Task<StorageScope> CreateAsync([CallerMemberName] string cal
         }
 
         /// <summary>
-        ///   Performs the tasks needed to create a new Event Hubs namespace within a resource group, intended to be used as
+        ///   Performs the tasks needed to create a new Azure storage account within a resource group, intended to be used as
         ///   an ephemeral container for the Event Hub instances used in a given test run.
         /// </summary>
         ///
-        /// <returns>The key attributes for identifying and accessing a dynamically created Event Hubs namespace.</returns>
+        /// <returns>The key attributes for identifying and accessing a dynamically created Azure storage account.</returns>
         ///
         public static async Task<StorageProperties> CreateStorageAccountAsync()
         {
             var subscription = TestEnvironment.EventHubsSubscription;
             var resourceGroup = TestEnvironment.EventHubsResourceGroup;
             var token = await ResourceManager.AquireManagementTokenAsync();
 
-            static string CreateName() => $"neteventhubs{ Guid.NewGuid().ToString("N").Substring(0, 12) }";
+            static string CreateName() => $"netstorage{ Guid.NewGuid().ToString("N").Substring(0, 12) }";
```

Comment:
nit:  Can we please revert this to `neteventhubs`?   This is the pattern that we've been asked to use for test resources.   It designates that the resource was created by the .NET Event Hubs test suite.